core/cli/package.json

```diff
@@ -20,17 +20,17 @@
     "test": "cd ../../ ; npx jest --silent --projects core/cli"
   },
   "devDependencies": {
-    "@dotcom-tool-kit/babel": "^3.2.0",
-    "@dotcom-tool-kit/backend-heroku-app": "^3.1.4",
-    "@dotcom-tool-kit/circleci": "^6.0.1",
-    "@dotcom-tool-kit/circleci-deploy": "^3.4.3",
-    "@dotcom-tool-kit/eslint": "^3.2.0",
-    "@dotcom-tool-kit/frontend-app": "^3.2.4",
-    "@dotcom-tool-kit/heroku": "^3.4.1",
-    "@dotcom-tool-kit/mocha": "^3.2.0",
-    "@dotcom-tool-kit/n-test": "^3.3.1",
-    "@dotcom-tool-kit/npm": "^3.3.1",
-    "@dotcom-tool-kit/webpack": "^3.2.0",
+    "@dotcom-tool-kit/babel": "4.0.0-beta.0",
+    "@dotcom-tool-kit/backend-heroku-app": "4.0.0-beta.0",
+    "@dotcom-tool-kit/circleci": "7.0.0-beta.0",
+    "@dotcom-tool-kit/circleci-deploy": "4.0.0-beta.0",
+    "@dotcom-tool-kit/eslint": "4.0.0-beta.0",
+    "@dotcom-tool-kit/frontend-app": "4.0.0-beta.0",
+    "@dotcom-tool-kit/heroku": "4.0.0-beta.0",
+    "@dotcom-tool-kit/mocha": "4.0.0-beta.0",
+    "@dotcom-tool-kit/n-test": "4.0.0-beta.0",
+    "@dotcom-tool-kit/npm": "4.0.0-beta.0",
+    "@dotcom-tool-kit/webpack": "4.0.0-beta.0",
```

Comment:
**question**: why are these `devDependencies`?

Reply:
they were probably only here for the integration test we used to have? we can remove them in a separate PR if they aren't needed anymore – good point!